modules/core/src/main/java/org/apache/ignite/internal/processors/cache/binary/BinaryMetadataFileStore.java

```diff
@@ -358,6 +358,24 @@ void prepareMetadataRemove(int typeId) {
         writer.prepareRemoveFuture(typeId);
     }
 
+    /** Dumps all binary meta files. */
```

Comment:
I just did what other such methods did, so it would look consistent. I think I remember that we had a "if it fits the do a one-liner" rule, but I don't think it was mandatory